src/graph.py

```diff
@@ -720,10 +723,12 @@ def augment_with_subcellular_localization(nxgraph):
         subcellular = pd.read_pickle(get_path('OmicsIntegrator', 'subcellular_compartments/subcellular.pickle'))
     except:
         # maybe need os.path.realpath(__file__)
-        subcellular = pd.read_pickle('../subcellular/subcellular.pickle')
-
-    nx.set_node_attributes(nxgraph, subcellular.loc[list(nxgraph.nodes())].dropna(how='all').to_dict(orient='index'))
+        subcellular = pd.read_pickle(os.path.dirname(os.path.realpath(__file__))+'/../subcellular/subcellular.pickle')
 
+    try:
+        nx.set_node_attributes(nxgraph, subcellular.loc[list(nxgraph.nodes())].dropna(how='all').to_dict(orient='index'))
```

Comment:
Previously, if none of the prizes were in the subcellular dict, it caused an error